pyproject.toml

```diff
@@ -7,6 +7,18 @@ requires = [
     "Cython>=3.0.0",
 ]
 
+[tool.cibuildwheel]
```

Comment:
I personally prefer putting all of these into the yaml file (just so they're all in one place) but that'll lead to duplicate lines. Not sure what people prefer.

Reply:
Putting commands which will only been seen by CI strikes me as a little strange. (Maybe this is because I haven't come across it in such a black-and-white case?). But I guess some options in this file in my other projects are effectively CI-only. I see the value in consolidating configs in one place!

Reply:
Moved them all into the build.yaml. The one thing that pushed me over the line is that we had two places to declare which python versions to use and which builds to skip, which is unnecessarily complicated.